abel/tests/test_direct.py

```diff
@@ -1,64 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I think that we should not add this line to the tests. They should not be made executable or nosetests will not run them. So just run then with `python <test_file.py>` instead of executing directly from bash.